src/Compilers/CSharp/Portable/Symbols/Source/SourceComplexParameterSymbol.cs

```diff
@@ -91,7 +120,7 @@ internal override ConstantValue ExplicitDefaultConstantValue
                 // Dev11 emits the first parameter as option with default value and the second as regular parameter.
                 // The syntactic default value is suppressed since additional synthesized parameters are added at the end of the signature.
 
-                return DefaultSyntaxValue ?? DefaultValueFromAttributes;
+                return OptionalParameter.ConstantValue ?? DefaultValueFromAttributes;
```

Comment:
>return OptionalParameter.ConstantValue ?? DefaultValueFromAttributes; [](start = 16, length = 69)

If we are covering all default values in all declarations, we probably should cover all arguments for all attributes as well. I.e. the following code shouldn't produce any warnings or errors:
```
class A : System.Attribute
{
    static int Age;

    [A(nameof(Age))]
    A(string Para1)
    { }
}
```